nevergrad/optimization/optimizerlib.py

```diff
@@ -860,6 +860,45 @@ def _internal_provide_recommendation(self) -> tp.ArrayLike:
         return self.avg
 
 
+class _Rescaled(base.Optimizer):
+    """Proposes a version of a base optimizer which works at a different scale."""
+    def __init__(
+            self,
+            parametrization: IntOrParameter,
+            budget: tp.Optional[int] = None,
+            num_workers: int = 1,
+            base_optimizer: base.OptCls = CMA,
+            scale: tp.Optional[float] = None,
+    ) -> None:
+        super().__init__(parametrization, budget=budget, num_workers=num_workers)
+        self.base_optimizer = base_optimizer(parametrization, budget=budget, num_workers=num_workers)
```

Comment:
It's better to share the same parametrization, that may be what caused the difference
```suggestion
        self.base_optimizer = base_optimizer(self.parametrization, budget=budget, num_workers=num_workers)
```

Also maybe rename to `self._optimizer`, the `base` is implicit + it shouldnt be directly accessible by the user